tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -5346,13 +5561,69 @@ Result D3D12Device::initialize(const Desc& desc)
                 m_features.add("half");
             }
         }
-        // Check double precision support
         {
             D3D12_FEATURE_DATA_D3D12_OPTIONS options;
             if (SLANG_SUCCEEDED(m_device->CheckFeatureSupport(D3D12_FEATURE_D3D12_OPTIONS, &options, sizeof(options))))
             {
+                // Check double precision support
                 if (options.DoublePrecisionFloatShaderOps)
                     m_features.add("double");
+
+                // Check conservative-rasterization support
+                auto conservativeRasterTier = options.ConservativeRasterizationTier;
+                if (conservativeRasterTier == D3D12_CONSERVATIVE_RASTERIZATION_TIER_3)
+                {
+                    m_features.add("conservative-rasterization-3");
```

Comment:
Unrelated to this PR: we probably need a systematic way to derive the names for features, and probably even want to have a `gfx`-internal representation of features that is richer than just strings.

Ideally we would have something like:

```
// in gfx.h

// A single enumeration that lists all the optional GPU API
// features that gfx understands, including both multi-API
// and API-specific features.
//
enum class FeatureID : int32_t { ... }

// We don't have this today, but probably need it at some
// point:
//
// A connection to the gfx library/implementation (gfx.dll)
struct IInstance : ISlangUnknown
{
    const char* getFeatureName(FeatureID id) = 0;
    FeatureID findFeatureByName(const char* name) = 0;
    ...
};
```

```
// Inside the implementation:

struct FeatureInfo
{
    // the id number and name for the feature
    FeatureID id;
    const char* name;

    // Optional: the other features that this feature depends on
    // or implies the existence of. Allows recursively walking all
    // the features transitively implied by a starting-point feature,
    // without having to encode that information as code.
    int requireCount;
    FeatureID const* requires;
};
static const FeatureInfo kFeatures[] = { ... };
```

For API-specific features, we'd want to have ones named very closely after the API-specific ones:

```
FeatureID::D3D12_CONSERVATIVE_RASTERIZATION_TIER_2
FeatureID::VK_some_vulkan_thing
```

For the API-specific ones, I'd argue that the string names should probably match the enumerant exactly, *especially* for Vulkan where string names for extensions are already needed.

Then we'd also want to have ones based on our own naming scheme that unifies across APIs:

```
FeatureID::CrossPlatformFeature
```

For this case it is harder to decide if we should have the name `cross-platform-feature` or have the external name also be `CrossPlatformFeature` for consistency... No string opinion right now.

Anyway, none of this really has anything to do with this PR.